www-servers/automx2/files/init

```diff
@@ -0,0 +1,26 @@
+#!/sbin/openrc-run
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+: ${AUTOMX2_CONF:="/etc/${RC_SVCNAME}.conf"}
+: ${AUTOMX2_USER:="automx2"}
+: ${AUTOMX2_ARGS:="--port 4243"}
+
+command="/usr/bin/python"
```

Comment:
I'm gonna bet this will be python3.6 on 99 % of systems even though your ebuild "only works with python3.7". Maybe you need to fix shebang on ebuild for this?

Reply:
What shebang are you referring to? The init script launches Python, which launches Flask, which launches automx2.

We've spent quite some time trying to figure out how to enforce a particular Python runtime version, asking the Python team for feedback, and what we came up with is what you see. If you have a better solution, I'd be interested to hear it.